package.json

```diff
@@ -87,6 +87,7 @@
     "@vitejs/plugin-vue": "^4.3.4",
     "@vue/eslint-config-prettier": "^8.0.0",
     "@vue/eslint-config-typescript": "^12.0.0",
+    "@vue/test-utils": "^2.4.3",
```

Comment:
Please downgrade to 2.4.1 as this version is for latest versions of vite (5 vs current 4) and vitest (1 vs 0.x)